src/App.tsx

```diff
@@ -1,27 +1,47 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './App.scss';
 import { GoodsList } from './GoodsList';
+import { getAll, get5First, getRed } from './api/goods';
+import { Good } from './types/Good';
 
-// import { getAll, get5First, getRed } from './api/goods';
-// or
-// import * as goodsAPI from './api/goods';
+export const App: React.FC = () => {
+  const [goods, setGoods] = useState<Good[]>([]);
 
-export const App: React.FC = () => (
-  <div className="App">
-    <h1>Dynamic list of Goods</h1>
+  const handleGoods = async (getMethod: any) => {
```

Comment:
Don't use any. Try to specify the type. And the `getMethod` isn't meaningful enough. Try to rename it (getGoods, maybe)
<img width="575" alt="image" src="https://user-images.githubusercontent.com/99403198/230716813-0edfa8b6-0fc5-489e-8188-00f05a12b858.png">